docs/book/src/clusterctl/commands/alpha-topology-plan.md

```diff
@@ -424,6 +424,14 @@ or validation on these objects. Defaulting and validation is only run on Cluster
 
 </aside>
 
+<aside class="note warning">
+
+<h1>Contract compatibility</h1>
+
+All the API versions of resources in the input are assumed to be compatible with the current version of Cluster API contract.
```

Comment:
Could we maybe expand this with an example. It's not obvious to me what the limitiations are from this sentence alone.

Reply:
I reworded the note to talk about the version and the contract compatibility. Please take a look and let me know if it is still not clear.

Reply:
Looks good!